### ISSUE TITLE
🚧 🐞 Bug Report: invalid HTML element in list view

### ISSUE THREAD

**Issue by baldusmarketing** [elementor/elementor#20747]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

We use the Encyclopedia plugin by Dennis Hope. 
According to the developer, it is due to Elementor because the generated HTML code is invalid. Can you please check how to fix this? The Elementor support could not help me and referred me to this group. Thank you for your support. 

### Steps to reproduce

1. Aktivate Encylopedia Plugin by Dennis Hoppe
2. List is displayed incorrectly.
3. all proposed scenarios according to ELementor support have not brought about any improvement

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.20.2
	MySQL version: MariaDB Server v10.3.31
	PHP Version: 7.4.33
	PHP Memory Limit: 128M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://baldus-sedation.de
	Home URL: https://baldus-sedation.de
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: de-DE
	Timezone: Europe/Berlin
	Admin Email: m.schenk@baldus-sedation.de
	Debug Mode: Inactive

== Theme ==
	Name: Astra Child
	Version: 3.7.5.1637676588
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 3.9.4
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: de_DE_formal
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36

== Active Plugins ==
	ActiveCampaign
		Version: 8.1.10
		Author: ActiveCampaign

	Advanced Custom Fields
		Version: 6.0.6
		Author: WP Engine

	Anti-Spam by CleanTalk
		Version: 6.0.1
		Author: СleanTalk <welcome@cleantalk.org>

	Astra Pro
		Version: 3.9.4
		Author: Brainstorm Force

	Borlabs Cookie - Cookie Opt-in
		Version: 2.2.58
		Author: Borlabs GmbH

	Code Snippets
		Version: 3.2.2
		Author: Code Snippets Pro

	Disable Variable Product Price Range Woocommerce
		Version: 1.8
		Author: Geek Code Lab

	Duplicate Page
		Version: 4.4.9
		Author: mndpsingh287

	Elementor
		Version: 3.9.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.0
		Author: Elementor.com

	EWWW Image Optimizer
		Version: 6.9.2
		Author: Exactly WWW

	Font Awesome
		Version: 4.3.1
		Author: Font Awesome

	Happy Elementor Addons
		Version: 3.7.2
		Author: weDevs

	Health Check & Troubleshooting
		Version: 1.5.1
		Author: The WordPress.org community

	LuckyWP Table of Contents
		Version: 2.1.4
		Author: LuckyWP

	Modern Events Calendar Lite
		Version: 6.5.6
		Author: Webnus

	OMGF
		Version: 5.5.0
		Author: Daan from Daan.dev

	PayPal PLUS for WooCommerce
		Version: 2.2.1
		Author: Inpsyde GmbH

	Premium Multi Scroll for Elementor
		Version: 1.0.5
		Author: Leap13

	Rank Math SEO
		Version: 1.0.103.1
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.25
		Author: Rank Math

	Redirector
		Version: 3.0.1
		Author: Ralf Hortt

	Simple User Avatar
		Version: 4.0
		Author: Matteo Manna

	Thrive Product Manager
		Version: 1.5.1
		Author: Thrive Themes

	Thrive Quiz Builder
		Version: 3.13.1
		Author: <a href="https://thrivethemes.com">Thrive Themes</a>

	Tidio Chat
		Version: 6.0.2
		Author: Tidio Ltd.

	TranslatePress - Developer
		Version: 1.2.2
		Author: Cozmoslabs, Razvan Mocanu, Madalin Ungureanu

	TranslatePress - Multilingual
		Version: 2.4.3
		Author: Cozmoslabs, Razvan Mocanu, Madalin Ungureanu, Cristophor Hurduban

	Web 2.0 Directory plugin
		Version: 2.9.15
		Author: salephpscripts.com

	WooCommerce
		Version: 7.2.0
		Author: Automattic

	WooCommerce Google Analytics Integration
		Version: 1.5.17
		Author: WooCommerce

	WooCommerce orgaMAX Shop API
		Version: 3.8.0
		Author: Albert Brückmann

	Woocommerce Single Product Page Customizer
		Version: 2.3
		Author: Geek Code Lab

	WooCommerce Table Rate Shipping
		Version: 3.0.29
		Author: WooCommerce

	WPForms
		Version: 1.7.2
		Author: WPForms

	WPForms Conversational Forms
		Version: 1.6.0
		Author: WPForms

	WPForms Form Pages
		Version: 1.4.1
		Author: WPForms

	WP Mail SMTP
		Version: 3.6.1
		Author: WPForms

	WP Rocket
		Version: 3.12.3.1
		Author: WP Media

== Must-Use Plugins ==
	Health Check Troubleshooting Mode
		Version: 1.8.0
		Author: 

== Elementor-Experimente ==
	Optimierte DOM Ausgabe: Aktiv
	Verbessertes Laden von Assets: Aktiv
	Verbessertes Laden von CSS: Aktiv
	Inline-Schriftarten-Symbole: Standardmäßig deaktiviert
	Verbesserungen der Zugänglichkeit: Standardmäßig aktiviert
	Zusätzliche benutzerdefinierte Breakpoints: Aktiv
	Import Export Website Kit: Standardmäßig aktiviert
	Native WordPress-Widgets aus den Suchergebnissen ausblenden: Standardmäßig aktiviert
	admin_menu_rearrangement: Standardmäßig deaktiviert
	Flexbox Container: Aktiv
	Default to New Theme Builder: Standardmäßig aktiviert
	Startseiten: Standardmäßig aktiviert
	Farbpalette: Standardmäßig aktiviert
	Bevorzugte Widgets: Standardmäßig aktiviert
	Admin Leiste: Aktiv
	Lazy Load Background Images: Standardmäßig deaktiviert
	Save as Default: Standardmäßig aktiviert
	Page Transitions: Standardmäßig aktiviert
	Notes: Inaktiv
	Loop: Standardmäßig deaktiviert
	Form Submissions: Standardmäßig aktiviert
	Scroll Snap: Standardmäßig aktiviert

== Protokoll ==
	
Log: showing 20 of 332022-11-14 07:50:39 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-11-14 07:50:39 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-11-14 07:50:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.0',
  'to' => '3.8.1',
)]
2022-12-02 16:48:32 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.1',
  'to' => '3.8.2',
)]
2022-12-02 16:48:34 [info] elementor-pro::elementor_pro_updater Started 
2022-12-02 16:48:34 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-12-02 16:48:34 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-12-02 16:48:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.1',
  'to' => '3.8.2',
)]
2022-12-07 07:48:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-07 07:48:31 [info] elementor::elementor_updater Started 
2022-12-07 07:48:31 [info] Elementor/Upgrades - _on_each_version Start  
2022-12-07 07:48:31 [info] Elementor/Upgrades - _on_each_version Finished 
2022-12-07 07:48:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-07 07:48:34 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-07 19:49:40 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-07 19:49:42 [info] elementor-pro::elementor_pro_updater Started 
2022-12-07 19:49:42 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-12-07 19:49:42 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-12-07 19:49:42 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-07 19:49:44 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]

JS: showing 6 of 6JS: 2022-11-03 07:56:10 [error X 4][https://baldus-sedation.de/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.0:3:1061530] Cannot read properties of null (reading \'getBoundingClientRect\') 
JS: 2022-11-03 08:12:57 [error X 17][https://baldus-sedation.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:51824] Cannot read properties of undefined (reading \'replace\') 
JS: 2022-11-03 10:27:27 [error X 10][https://baldus-sedation.de/wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.8.0:2:2932] elementor_new_template_form_controls is not defined 
JS: 2022-11-09 13:39:40 [error X 2][https://baldus-sedation.de/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.8.0:2:19453] Cannot read properties of undefined (reading \'top\') 
JS: 2022-11-28 12:21:33 [error X 9][https://baldus-sedation.de/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:288289] Cannot read properties of undefined (reading \'id\') 
JS: 2022-12-01 22:25:30 [error X 1][https://baldus-sedation.de/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:1061528] $ is null 

PHP: showing 4 of 4PHP: 2022-11-04 10:59:01 [error X 4][/var/www/vhosts/baldus-sedation.de/httpdocs/wp-content/plugins/elementor/core/common/modules/finder/module.php::90] Uncaught Error: Call to a member function get_category_items() on array in /var/www/vhosts/baldus-sedation.de/httpdocs/wp-content/plugins/elementor/core/common/modules/finder/module.php:90
Stack trace:
#0 [internal function]: Elementor\Core\Common\Modules\Finder\Module->ajax_get_category_items()
#1 /var/www/vhosts/baldus-sedation.de/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func()
#2 /var/www/vhosts/baldus-sedation.de/httpdocs/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#3 /var/www/vhosts/baldus-sedation.de/httpdocs/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#4 /var/www/vhosts/baldus-sedation.de/httpdocs/wp-includes/plugin.php(517): WP_Hook->do_action()
#5 /var/www/vhosts/baldus-sedation.de/httpdocs/wp-admin/admin-ajax.php(188): do_action()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-09 13:22:16 [notice X 4][/var/www/vhosts/baldus-sedation.de/httpdocs/wp-content/plugins/elementor-pro/modules/woocommerce/classes/products-renderer.php::146] Trying to get property 'taxonomy' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-12-05 09:30:42 [notice X 1][/var/www/vhosts/baldus-sedation.de/httpdocs/wp-content/plugins/elementor-pro/modules/forms/classes/activecampaign-handler.php::133] Undefined index: checkbox [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-12-07 09:45:01 [notice X 1][/var/www/vhosts/baldus-sedation.de/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Unkompatibel
	Happy Elementor Addons: Unkompatibel
	Premium Multi Scroll for Elementor: Unkompatibel

== Elementor Pro - Compatibility Tag ==
	
	Happy Elementor Addons: Unkompatibel
	Premium Multi Scroll for Elementor: Unkompatibel

**Comment by nicholaszein** [comment]
Hello @baldusmarketing,

Thank you for submitting your issue! 🙏

### ❌ While reviewing your report, we noticed it was missing some key pieces of information:

- **Description:** In your description, you need to explain in detail what happened, as you would explain to someone that doesn't know anything about your website or Elementor. Give as much information as possible. Explain where your issue is happening, and what is the actual issue, share screenshots, and screen recordings if needed. This is important information that will help solve your issue quicker;
- **Steps to replicate:** Think of it as explaining to someone over the phone how to replicate your issue. They can't see the problem, so explain it in detail in a way that we can try to replicate your issue in our own environments. Be detailed, explain how you've built your page, and what steps you've taken to reach the issue you've found. Give us a path to follow so we can replicate your issue in our environments and understand what's going on;
- **Isolating the Problem:** You need to make sure this only happens while using the latest versions of Elementor, Elementor Pro, and the Hello Elementor theme. We do not provide support for 3rd-party themes and plugins.

Without them, we simply can't help. 😕 

Best regards

**Comment by nicholaszein** [comment]
Hello there @baldusmarketing, 

#### 🕙 It's been a while since we answered you, and we haven't received any feedback. 

## 🚧 For this reason, we are closing this issue because of no activity.

👉 If you wish to continue this discussion, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

Feel free to submit a new issue in accordance with these guidelines. 😉 

Kind regards
